### PR TITLE
Add a screenshot of the GitHub branch check setting for clarity.

### PR DESCRIPTION
**Pull request by adrcunha**

### CODE REVIEW

File: ci/prow/prow_setup.md [hunk 1]
```diff
@@ -39,40 +47,55 @@
...
+   (ask one of the owners of knative/test-infra).
 
-1. Wait a few minutes, check that Prow is working by entering `/woof` as a comment in any PR in the new repo.
+1. Wait a few minutes, check that Prow is working by entering `/woof` as a
+   comment in any PR in the new repo.
 
 1. Set **tide** as a required status check for the master branch.
 
-### Setting up jobs for a new repo 
+   ![Branch Checks](branch_checks.png)
```
Comment (chaodaiG) [inline ci/prow/prow_setup.md]: this doesn't seem like belong to here

Comment (adrcunha) [reply ci/prow/prow_setup.md]: It does. Why you think it doesn't?

Comment (chaodaiG) [reply ci/prow/prow_setup.md]: I saw the same pic showed up at line 101 also, is this intended?

Comment (adrcunha) [reply ci/prow/prow_setup.md]: Yes. Same setting, different checks.

**Pull request merged**